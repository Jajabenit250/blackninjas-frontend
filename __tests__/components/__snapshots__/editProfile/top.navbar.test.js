import React from 'react';
import { shallow } from 'enzyme';
import TopNavBar from '../../../../src/components/profile/top.nav.bar';

describe('Top navigation bar component', () => {
	it('should render the top navigation bar  component successfully', () => {
		const wrapper = shallow(<TopNavBar />);
		expect(wrapper).toMatchSnapshot();
	});
});
