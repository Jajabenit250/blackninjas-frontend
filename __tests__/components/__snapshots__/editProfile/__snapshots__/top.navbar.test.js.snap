// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top navigation bar component should render the top navigation bar  component successfully 1`] = `
<Styled(MuiBox)
  display="flex"
  justifyContent="flex-end"
  width={1}
>
  <Styled(MuiBox)
    pr={1}
    style={
      Object {
        "marginTop": "10px",
      }
    }
  >
    <img
      height="24px"
      src="https://res.cloudinary.com/dby88h516/image/upload/v1580893608/email_1_jupvlq.svg"
      width="100%"
    />
  </Styled(MuiBox)>
  <Styled(MuiBox)
    pl={1}
    style={
      Object {
        "marginTop": "10px",
      }
    }
  >
    <img
      height="24px"
      src="https://res.cloudinary.com/dby88h516/image/upload/v1580892708/bell_1_culslv.svg"
      width="100%"
    />
  </Styled(MuiBox)>
</Styled(MuiBox)>
`;
